### PR TITLE
Second phase of stabilization docs formatting, spelling, and updates

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview
Updates formatting and spelling in various documentation. Continuation of https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6661

### CODE REVIEW

File: Documentation/Contributing/Feature_Contribution_Process.md [hunk 1]
```diff
@@ -58,51 +59,51 @@ If needed, the architectural document can be updated to align with any changes t
...
 
-## Manager Implementation Requirements
+### Manager Implementation Requirements
 
 * Assembly Definitions for code outside of the `MixedRealityToolkit/_Core` folder.
-    * This ensures features are self-contained and have no dependencies to other features.
-    * This only applies to `MixedRealityToolkit` folder.
+  * This ensures features are self-contained and have no dependencies to other features.
+  * This only applies to `MixedRealityToolkit` folder.
 * Be defined using an interface found in `MixedRealityToolkit/_Core/definitions/<FeatureName>System`.
```
Comment (davidkline-ms) [inline Documentation/Contributing/Feature_Contribution_Process.md]: folder names are different now (ex: there is no _Core folder any longer)

File: Documentation/Contributing/Feature_Contribution_Process.md [hunk 2]
```diff
@@ -58,51 +59,51 @@ If needed, the architectural document can be updated to align with any changes t
...
 * A feature's concrete manager implementation should setup and verify scene is ready for that system to use in `Initialize`.
 * A feature's concrete manager should also clean up after themselves removing anything created in the scene in `Destroy`.
 * Be registered with the Mixed Reality Manager.
-    * If the feature is a core feature, this should be hard coded into the `MixedRealityManager` and added to the `MixedRealityConfigurationProfile`.
-        * This includes being able to specify a concrete implementation via dropdown using `SystemType`.
-        * Features should have a configuration profile that derives from a scriptable object.
-        * A default configuration profile located in `MixedRealityToolkit-SDK/Profiles` and be assigned in the default configuration profile for the Mixed Reality Manager
-    * If this feature is **not** a core feature, then it must be registered using the component configuration profile and implement `IMixedRealityComponent`.
+  * If the feature is a core feature, this should be hard coded into the `MixedRealityManager` and added to the `MixedRealityConfigurationProfile`.
```
Comment (davidkline-ms) [inline Documentation/Contributing/Feature_Contribution_Process.md]: MixedRealityManager -> MixedRealityToolkit and CoreServices

File: Documentation/Contributing/Feature_Contribution_Process.md [hunk 3]
```diff
@@ -1,10 +1,11 @@
-10/1/2019: This page is deprecated because it provides guidelines for contributing very large systems to MRTK before the 2.0 release. After the 2.0 release, large changes need to be performed more carefully, and the process for this is not yet decided. We expect most MRTK contributions to have much smaller changes than what is covered here.
-
 # Feature Contribution Process
 
+> [!WARNING]
```
Comment (davidkline-ms) [inline Documentation/Contributing/Feature_Contribution_Process.md]: Perhaps it is time to refresh / remove this article as it is out of date, and as this comment states, large changes need to be handled carefully.

File: Documentation/Contributing/Feature_Contribution_Process.md (hunk 1 above)
Comment (keveleigh) [reply Documentation/Contributing/Feature_Contribution_Process.md]: Oh right! I didn't go through these for content so much as formatting/spelling, but I'll take a quick pass now,

Comment (keveleigh) [reply Documentation/Contributing/Feature_Contribution_Process.md]: Updated

File: Documentation/Contributing/Feature_Contribution_Process.md (hunk 2 above)
Comment (keveleigh) [reply Documentation/Contributing/Feature_Contribution_Process.md]: Updated

File: Documentation/Contributing/Feature_Contribution_Process.md (hunk 3 above)
Comment (keveleigh) [reply Documentation/Contributing/Feature_Contribution_Process.md]: I did a quick pass on refreshing this article. I'm not against replacing it with a new doc in an upcoming iteration

**Pull request merged**